<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>snake</class>
 <widget class="QWidget" name="snake">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>10</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 69, 7);
font: 75 9pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>积分</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_mark">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>50</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>100</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 65, 2);
font: 75 9pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>存活时长</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_time">
   <property name="geometry">
    <rect>
     <x>583</x>
     <y>132</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_star">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>220</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_stop">
   <property name="geometry">
    <rect>
     <x>581</x>
     <y>277</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_restar">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>330</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>重开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Next_but">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>390</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>下一关</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
